sdk-extensions/autoconfigure/src/test/java/io/opentelemetry/sdk/autoconfigure/ConfigPropertiesTest.java

```diff
@@ -216,4 +218,31 @@ void durationUnitParsing() {
                 .getDuration("duration"))
         .isEqualTo(Duration.ofMillis(8));
   }
+
+  @Test
+  void defaultMethods() {
+    ConfigProperties properties = DefaultConfigProperties.get(emptyMap());
+    assertThat(properties.getBoolean("foo", true)).isTrue();
```

Comment:
Noticed it looks like there's missing test coverage that the default methods will return the present value, these are just checking the fallback